pkg/products/monitoring/prometheusRules.go

```diff
@@ -151,6 +151,26 @@ func (r *Reconciler) reconcileKubeStateMetricsAlerts(ctx context.Context, server
 			For:    "15m",
 			Labels: map[string]string{"severity": "critical"},
 		},
+		{
+			Alert: "KubePodOutOfMemoryKilledWarning",
+			Annotations: map[string]string{
+				"sop_url": "TBA",
+				"message": "Pod {{ $labels.namespace }} / {{  $labels.pod  }} has been killed due to being Out Of Memory.",
```

Comment:
Should the message be the same on both these alerts?

Reply:
Yes, that's a remainder from the previous approach that the first query had, something closer to `Pods in {{ $labels.namespace }} are being killed off due to OOM` would probably fit better, will change in the next commit :+1: